tools/deb/postinst.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+action="$1"
+oldversion="$2"
+
+umask 022
+
+if ! getent passwd istio >/dev/null; then
+    addgroup --system istio
+    adduser --system --group --home /var/lib/istio istio
+fi
+
+if [ ! -e /etc/istio ]; then
+   # Backward compat.
+   ln -s /var/lib/istio /etc/istio
+fi
+
+mkdir -p /var/lib/istio/envoy
```

Comment:
deb don't have the notion of an install prefix that we're supposed to use ? or variables for var/lib etc ?